pages/documentation/guidance/performance/glossary.md

```diff
@@ -250,15 +250,15 @@ Time to interactive can be measure in one of two ways:
 
 ##### Lighthouse
 
-[Lighthouse], the Chrome plugin and command line testing tool, includes time to interactive as one of the metrics it tests. To test, run either the plugin on your site or the [CLI tool](https://developers.google.com/web/tools/lighthouse/#cli) (as explained on the Google Tools for Web Developers page) with the appropriate options.
+[Lighthouse], the Chrome plugin and command line testing tool, includes time to interactive as one of the metrics it tests. To test, run either the plugin on your site or the [CLI tool](https://developer.chrome.com/docs/lighthouse/overview/#cli) (as explained on the Chrome for developers page) with the appropriate options.
```

Comment:
@amyleadem @mahoneycm - Can you check this link as well just to confirm this is the correct new URL for the intended page? Received a new test failure for this link. Also a couple naming changes.

Reply:
> [!note]
> Confirming content is the same as the old url. [Before](https://web.archive.org/web/20220201003240/https://developers.google.com/web/tools/lighthouse/#cli) | [After](https://developer.chrome.com/docs/lighthouse/overview/#cli)

Reply:
Future improvement and possibly a quick content change (cc @finekatie) — can we reduce the repetition of "as explained on the Chrome for developers" mentions here? 

Not blocking to this at all, just would improve understandability

Reply:
I see what you're talking about @annepetersen . It seems like users are already being taken to the Chrome for developers site from other links in the same paragraphs, so I'm not sure why it needs to be explained to them, unless there is concern that a first-time user won't be familiar with the site. I wonder if we could have one sentence somewhere that mentions Chrome for developers as a resource, so that we don't have to get into it in each section. 
I regards to snapshot-in-time, I notice the page doesn't have a changelog. Would adding one help with that issue?